jenkins_remote/jenkins_remote.py

```
@@ -0,0 +1,145 @@
+#!/usr/bin/env python3
+
+import argparse
+import requests
+import sys
+import os
+
+JENKINS_URL = (
+    "https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job"
+)
+
+OCP_COMMON_JOBS = "ocp-common/job"
+CI_MONITOR_JOBS = "ci_monitor/job"
+
+
+def destroy_cluster(args):
+    print(f"destroying cluster {args.build_number}")
+    job_url = "Flexy-destroy"
+    params = f"BUILD_NUMBER={args.build_number}"
+    destroy_job_url = (
+        f"{JENKINS_URL}/{OCP_COMMON_JOBS}/{job_url}/buildWithParameters?{params}"
+    )
+    r = requests.post(
+        destroy_job_url,
+        auth=(os.getenv("USER"), os.getenv("JENKINS_API_TOKEN")),
+        verify="/etc/certs/ipa.crt",
+    )
+    if r.status_code == 201:
+        nbuild = _get_jenkins_build_number(r.headers["location"])
+        print("triggered job: {nbuild}  to destroy cluster successfully")
+
+
+def ci_monitor(args):
+    runid = args.run_id
+    toFileJira = args.file_jira
+    print(f"running ci monitor for run id {runid} with file_jira = {toFileJira} ")
+    job_url = "ci_failure_summary"
+    params = f"TEST_RUN_ID={runid}&FILE_JIRA_ISSUES={toFileJira}"
+    ci_monitor_url = (
+        f"{JENKINS_URL}/{CI_MONITOR_JOBS}/{job_url}/buildWithParameters?{params}"
+    )
+    r = requests.post(
+        ci_monitor_url,
+        auth=(os.getenv("USER"), os.getenv("JENKINS_API_TOKEN")),
+        verify="/etc/certs/ipa.crt",
+    )
+    if r.status_code == 201:
+        nbuild = _get_jenkins_build_number(r.headers["location"])
+        print(f"triggered job: {nbuild} to report CI failures successfully")
```

Maybe raise an exception here if you don't get a 201 status code back?